build-frontend.sh

```diff
@@ -5,4 +5,7 @@ set -e
 pushd frontend
 yarn install
 yarn run build
+pushd integration-tests-cypress/cypress-e2e
```

Comment:
Did you mean to nest this?

Reply:
```bash 
pushd frontend
yarn install
yarn run build
popd
pushd frontend/integration-tests-cypress/cypress-e2e
yarn install
popd
```

Reply:
Not a huge deal, this doesn't change often

Reply:
fixed